src/navigation/Menu/Menu.spec.tsx

```diff
@@ -0,0 +1,222 @@
+      },
+      {
+        id: 2,
+        value: <ButtonDisplay label="Add" icon={<AddIcon />} />,
+      },
+      {
+        id: 3,
+        value: <ButtonDisplay label="Delete" icon={<DeleteIcon />} />,
+      },
+    ],
+  },
+  {
+    id: 91,
+    name: "End",
+    value: [
+      {
+        id: 4,
+        value: <ButtonDisplay label="Print" icon={<FilesIcon />} />,
+      },
+      {
+        id: 5,
+        value: <ButtonDisplay label="Move" icon={<Drag />} />,
+      },
+      {
+        id: 6,
+        value: <ButtonDisplay label="Delete" icon={<DeleteIcon />} />,
+      },
+    ],
+  },
+];
+
+const menuSepratorItems = [
+  {
+    id: 91,
+    value: [
+      {
+        id: 4,
+        value: <ButtonDisplay label="Print" icon={<FilesIcon />} />,
+      },
+      {
+        id: 5,
+        value: <ButtonDisplay label="Move" icon={<Drag />} />,
+      },
+      {
+        id: 6,
+        value: <Divider />,
+      },
+      {
+        id: 7,
+        value: <ButtonDisplay label="Upload file" icon={<UploadIcon />} />,
+      },
+      {
+        id: 8,
+        value: <ButtonDisplay label="Sort it" icon={<SortDown />} />,
+      },
+    ],
+  },
+];
+
+const menuSubMenuItems = [
+  {
+    id: 92,
+    name: "Menu bottom",
+    value: [
+      {
+        id: 6,
+        value: <ButtonDisplay label="Move" icon={<Drag />} />,
+      },
+      {
+        id: 7,
+        value: <ButtonDisplay label="Print" icon={<FilesIcon />} />,
+      },
+      {
+        id: 8,
+        value: (
+          <Button
+            radius="none"
+            kind="text"
+            block
+            align="left"
+            textStyle="normal"
+            size="small"
+            leadingIcon={<UploadIcon />}
+            variant="dark"
+          >
+            Change
+          </Button>
+        ),
+        children: [
+          { id: 100, value: <ButtonDisplay label="Crop" /> },
+          { id: 101, value: <ButtonDisplay label="Cut" /> },
+          { id: 102, value: <ButtonDisplay label="Copy" /> },
+        ],
+      },
+    ],
+  },
+];
+
+describe("Menu", () => {
+  it("Should show a Menu", () => {
+    render(<Menu menuItems={simpleMenuItems} />);
+    const menu = screen.getByRole("menu");
+    const button = screen.getByText(/Open/i);
+    const checkboxes = screen.getAllByRole("menuitem");
+
+    expect(menu).toBeInTheDocument();
+    expect(button).toBeInTheDocument();
+    expect(checkboxes.length).equal(3);
+  });
+
+  it("Should show a Menu with two sections with title", () => {
+    render(<Menu menuItems={doubleMenuTitleItems} />);
+    const titleSection = screen.getByRole("group", { name: /Top/i });
+    const sections = screen.getAllByRole("group");
+    const button = screen.getByText(/Open/i);
+
+    expect(titleSection).toBeInTheDocument();
+    expect(button).toBeInTheDocument();
+    expect(sections.length).equal(2);
+  });
+
+  it("Should show a Menu with separator", () => {
```

Comment:
typo : 
-> "Should show a Menu with a separator"
-> "Should display a Menu including a separator"

Reply:
modified